VTEX - Promotions & Taxes API.json

```diff
@@ -91,7 +91,7 @@
                                         "quantity": {
                                             "type": "integer",
                                             "title": "quantity",
-                                            "description": "Quantity of coupons created",
+                                            "description": "Quantity of coupons created. The quantity number must be a minimum of `1` and at maximum `1000`. If the amount is `1`, the coupon code will be the same as the request. If the quantity is greater than `1`, the coupon code will receive a random suffix.",
```

Comment:
```suggestion
                                            "description": "Quantity of coupons created. The value must be a minimum of `1` and at maximum `1000`. If the amount is `1`, the coupon code will be the same as the request. If the value is greater than `1`, the coupon code will receive a random suffix.",
```

Reply:
`same as the request` Como assim?

Reply:
Você pode colocar no request body um `couponCode` e a quantidade um, ele vai criar um cupom com o `couponCode` dado na request. Caso essa quantidade seja maior que um, serão gerados `couponCode`s aleatórios